FortranCon2021-stdlib/stdlib-talk.tex

```diff
@@ -0,0 +1,302 @@
+b2 = [(.true., i=1,6)]        ! S6B111111
+
+call xor(b1, b2)              ! S6B110001, S6B111111
+
+call b1%set(2, 4)             ! S6B111111 -- N.B. 0-based index
+print *, b1 == b2             ! T
+end
+\end{verbatim}
+  \end{block}
+
+  \note{
+    The second demo is of the bitsets module.
+    This provides a feature that is often proposed for standardization: an abstract type for working with strings of bits.
+    The basic bitset type uses a 64-bit integer internally, but it keeps track of the actual length of the bitset being represented.
+    Here, I'm initializing two 6-bit sets, one from a string and one from an array of logicals.
+    The comment shows a string representation of the bitset, which can be taken as a bitset 'literal' for the purposes of I/O.
+    Here, the prefix 'S6B' means a set of 6 bits.
+    Most operations with bitsets work in-place rather than creating new instances.
+    For instance, all the usual boolean algebra operations -- like the exclusive-or here -- are implemented as subroutines that overwrite the first argument.
+    Likewise, when one sets or clears single bits or contiguous ranges of bits, the data is modified without making a copy.
+  }
+\end{frame}
+
+
+\begin{frame}
+  \frametitle{New contributors have been key to stdlib's growth}
+
+  \begin{itemize}
+  \item From 16 committors to 34, including our 2 GSoC students
+  \item Over 100 new Issues: bugs, workflow improvements, feature proposals
+  \end{itemize}
+
+  \begin{block}{Commits to stdlib since FortranCon 2020}
+    \centering
+    \includegraphics[width=\textwidth]{contrib.png}
+  \end{block}
+
+  \note{
+    So we got lots of great new functionality, and in large part that's thanks to an influx of new contributors to the stdlib project.
+    Not only has the number of modules doubles but so has the number of people who've committed changes to stdlib.
+    Included in that are two Google Summer of Code students Aman Godara and Chetan Karwa, who we will hear from shortly.
+    What that figure doesn't include is contributions made in the form of Issues and reviews of Pull Requests.
+    It's really great to see over 100 new Issues opened in the past year, since this not only includes bug reports but also discussion of workflow improvments and proposals for new features.
+    And looking at the timeline of commits since last year's FortranCon, we see a nice sustained development with big rushes of new development at the end of last summer and throughout this summer.
+  }
+\end{frame}
+
+
+\begin{frame}
+  \frametitle{stdlib is now easier to install}
+
+  \begin{itemize}
+  \item Dependencies
+    \begin{itemize}
+    \item Fortran compiler (supporting at least F2008)
+    \item CMake (or just make)
+    \item fypp preprocessor (python script)
+    \end{itemize}
+  \item Install each separately or use conda package manager
+  \item Exports both CMake package files \& pkg-config files
+  \item Support for fpm-based workflow is in progress
```

Comment:
fpm supports now stdlib using a specific branch of the stdlib project (since a couple of days).
See [here](https://github.com/fortran-lang/stdlib#build-with-fortran-langfpm) for moe details.

Reply:
Yes! I saw this shortly after opening this PR. I will update that slide accordingly. What great timing!